e2e/cmd/cmd.go

```diff
@@ -103,7 +103,12 @@ func newDeployCmd(def *app.Definition) *cobra.Command {
 		Use:   "deploy",
 		Short: "Deploys the e2e network",
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			_, err := app.Deploy(cmd.Context(), *def, cfg)
+			ctx := cmd.Context()
+			_, err := app.Deploy(ctx, *def, cfg)
+			if err != nil {
+				log.Error(ctx, "Deployment failed", err)
```

Comment:
Without this log, a misconfigured compose template would lead to a deployment failure without any error message in the console.

Reply:
As per go guidelines, don't "log AND return errors". We return the error below, so don't log it. The caller of this function will log if required.

Reply:
The caller being `libcmd.Main`

Reply:
Thanks a lot, this is very helpful! Will fix.